headers/addons/gif.h

```diff
@@ -0,0 +1,34 @@
+#ifndef GIF_H_
+#define GIF_H_
+
+#define GIF_ENABLED 0
+#define GIF_STARTING_GIF_ADDRESS 0
+#define GIF_NUMBER_OF_FRAMES 0
+
+#define GIF_EEPROM_DISTANCE 0xA000 // Leave room of 40kb for the EEPROM to grow
```

Comment:
we should not be manually modifying the EEPROM inside of an add-on. If we need to move around flash memory, this should either occur inside of the PROM library we're using, or in a main area. This can also potentially break protobuf.

Reply:
It's not modifying the simulated EEPROM it's using the unused memory between the EEPROM and where the code sits. One is at the start of the flash memory and the other is at the end. We are writing in middle using as high of as an address as possible while maintaining a safe distance from the EEPROM in case it increases in size, currently it's only using 16kb and I left 40kb more in case it needs to increase in size